src/ROUTES.js

```diff
@@ -17,7 +17,10 @@ export default {
     IOU_BILL: 'iou/split',
     SEARCH: 'search',
     SIGNIN: 'signin',
-    SET_PASSWORD_WITH_VALIDATE_CODE: 'setpassword/:validateCode',
+    SET_PASSWORD_WITH_VALIDATE_CODE: 'setpassword/:email/:validateCode',
```

Comment:
I would much rather we pass accountID here instead of email. This way we don't have to deal with special characters in the email breaking our code.